server/lib/configuration.js

```diff
@@ -346,6 +346,22 @@ const conf = module.exports = convict({
       'dist'
     ]
   },
+  pairing: {
+    clients: {
+      default: [
+        '3c49430b43dfba77', // Reference browser
+        'a2270f727f45f648', // Fenix
+      ],
+      doc: 'OAuth Client IDs that are allowed to pair. Remove all clients from this list to disable pairing.',
+      env: 'PAIRING_CLIENTS',
+      format: Array,
+    },
+    server_base_uri: {
+      default: 'wss://dev.channelserver.nonprod.cloudops.mozgcp.net',
```

Comment:
No, we don't put prod values in configuration.js AFAIS, but we can if you want. I put the production value in the deploy doc

Reply:
> No, we don't put prod values in configuration.js

I thought that's how we rolled unless the value was sensitive so that Ops didn't need to worry about updating their config.

Reply:
I can put the prod value in here, yeap 👍